apps/public-docsite-v9/src/Concepts/Theming.stories.mdx

```diff
@@ -0,0 +1,140 @@
+import { Meta, Source } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Developer/Theming" />
+
+# Theming
+
+The Fluent UI Theme is represented by a set of tokens. Each token resolves to a single value which can be assigned to a CSS property.
```

Comment:
````suggestion
The Fluent UI Theme is represented by a set of tokens. Each token resolves to a single value which can be assigned to a CSS property.

```tsx
const theme = {
  tokenA: 'var(--tokenA)',
  // etc.
}
```
````

Will it be helpful to show what it actually means?

Reply:
Sure, but the theme object is semantic name -> raw value, so I added an example of that. The later sections explain how that is added to DOM and consumed in styles.